src/librustdoc/html/render/mod.rs

```diff
@@ -1617,10 +1599,33 @@ fn render_impl(
     w.write_str(&close_tags);
 }
 
-fn render_impl_summary(
+fn render_rightside(
```

Comment:
Please add documentation on this function. :)

Reply:
Done 👍🏻

Reply:
Thanks!